_includes/google-analytics.html

```diff
@@ -4,8 +4,10 @@
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-  ga('create', '{{ site.google_analytics }}', 'auto');
+  ga('create', '{{ site.google_analytics.tracking_id }}', 'auto');
+  {% if site.google_analytics.anonymize_ip == true %}
+    ga('set', 'anonymizeIp', true);
+  {% endif %}
```

Comment:
@DirtyF @parkr that should work for most fields.  However, [some fields](https://devsite.googleplex.com/analytics/devguides/collection/analyticsjs/field-reference#create) can **only** be set in the `create` command, and some fields can **not** be set in the `create` command (they have to be set using the `set` command.

In other words, the following config wouldn't work with the above suggestion:

```yaml
site_speed_sample_rate: 10
anonymize_ip: true
transport: beacon
```